internal/liner/liner.go

```diff
@@ -24,13 +23,15 @@ func New(r io.Reader) *Liner {
 // Read reads a full line, automatically reading again if the line was not complete.
 // Each time an additional read is performed, doContinuation is called.
 // If the callback returns an error, the operation is aborted.
-func (l *Liner) Read(doContinuation func() error) ([]byte, map[string][]byte, error) {
+func (l *Liner) Read(doContinuation func() error) (string, error) {
```

Comment:
wouldn't be better to return `[]byte` ?

Reply:
we needed to convert to string for the C++ parser. This type has no other use other than to prepare the input for the parser.